system-tests/services/_scripts/external-volumes/volclean.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env python3
```

Comment:
This file concerns me because we have copied and pasted here, what happen when there is a change/update/security fix to the original file? we won't update this ever.

Can't we create a python file that import the original and uses it? That would be the ideal.